docs/usage/folders.md

```diff
@@ -195,16 +195,17 @@ folder = client.get_shared_item('https://app.box.com/s/gjasdasjhasd', password='
 Create a Shared Link
 --------------------
 
-A shared link for a file can be generated by calling
-[`file.get_shared_link(access=None, etag=None, unshared_at=None, allow_download=None, allow_preview=None, password=None)`][get_shared_link].
+A shared link for a folder can be generated by calling
+[`folder.get_shared_link(access=None, etag=None, unshared_at=None, allow_download=None, allow_preview=None,
+password=None, vanity_name=None)`][get_shared_link].
 This method returns a `unicode` string containing the shared link URL.
 
 <!-- sample put_folders_id add_shared_link -->
 ```python
-file_id = '11111'
+folder_id = '11111'
```

Comment:
Why did you replace the file shared link sample with a folder shared link sample?

Reply:
Because this is folder.md refering to examples for folder not files. The usage of the creating shared link for file is in file.md line 661.

Reply:
See https://github.com/box/box-python-sdk/pull/637/files#diff-eab6f6163ea11c58a349b415d9c099d9992646a2354271b809df3f81b14a496aR661